code/game/atoms_movable.dm

```diff
@@ -754,4 +754,92 @@
 	appearance = falltempoverlays[src]
 	falltempoverlays -= src
 	ignoreinvert = initial(ignoreinvert)
-	invisibility = init_invisibility
\ No newline at end of file
+	invisibility = init_invisibility
+
+/datum/proc/send_to_past(var/duration)
+	return
+
+/datum/var/being_sent_to_past = FALSE
+
+/atom/movable/send_to_past(var/duration)
+	var/current_loc = loc
+	var/list/resettable_vars = list(
+									"being_sent_to_past",
```

Comment:
It's because you used tabs.

And even if you didn't Jesus you only need to indent that once, not like 10×.